service/src/main/java/org/apache/griffin/core/metastore/hive/HiveMetaStoreServiceJDBCImpl.java

```diff
@@ -0,0 +1,224 @@
+package org.apache.griffin.core.metastore.hive;
+
+import org.apache.hadoop.hive.metastore.api.FieldSchema;
+import org.apache.hadoop.hive.metastore.api.StorageDescriptor;
+import org.apache.hadoop.hive.metastore.api.Table;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.cache.annotation.CacheConfig;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.stereotype.Service;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Author: qwang8
+ * Date:   2019-06-18.
+ */
+@Service
+@Qualifier(value = "hive_jdbc")
+@CacheConfig(cacheNames = "hive", keyGenerator = "cacheKeyGenerator")
```

Comment:
could you rename cache name as jdbchive?

Reply:
done